emperor/support_files/js/sceneplotview3d.js

```diff
@@ -162,13 +168,86 @@ define([
       [range.min[x], range.min[y], range.max[z]]
     ];
 
+    for (var i = 0; i < 3; i++){
```

Comment:
I prefer the loop, but I am willing to change it if you guys thinks that makes more sense.